csharp/Adapter/Microsoft.Spark.CSharp/Streaming/PairDStreamFunctions.cs

```diff
@@ -678,6 +683,18 @@ internal RDD<dynamic> Execute(double t, RDD<dynamic> stateRDD, RDD<dynamic> valu
 
             if (stateRDD == null)
             {
+                if (initialState != null)
```

Comment:
Yes. In the unit test https://github.com/Microsoft/Mobius/pull/472/files#diff-160b5c86e37ad78a112e816b47a4189dR275 an initialStateRdd instance is passed to `UpdateStateByKey`, and in https://github.com/Microsoft/Mobius/pull/472/files#diff-77f516db0b20256e81909b17713e9586L106 the `stateRdd` is set to null, so this block is covered in unit test.